Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/photo_field.tsx

```diff
@@ -0,0 +1,653 @@
+import { css, StyleSheet } from 'aphrodite/no-important';
+import * as React from 'react';
+import { Transition } from 'react-transition-group';
+import { DisplaySize, HBoxLayout, HLine, Padding, VBoxLayout } from '../../..';
+
+/** Various the mode that the PhotoField can be displayed at. */
+export enum DisplayMode {
+
+  /** Only the photo is visible. */
+  Display,
+
+  /** The uploader is visible. */
+  Uploading
+}
+
+interface Properties {
+
+  /** Determines the size to render the component at. */
+  displaySize: DisplaySize;
+
+  /** Determines if the ChangePictureModal is visible or not. */
+  displayMode: DisplayMode;
+
+  /** The URL the image is located at. */
+  imageSource?: string;
+
+  /** Determines if the image can be changed or not. */
+  readonly?: boolean;
+
+  /** A value from 0 to 200 that determines how zoomed in the image will be. */
+  scaling: number;
+
+  /** Callback to hide or show the uploader. */
+  toggleUploader: () => void;
+
+  /** Callback to store the file and the scaling for the file. */
+  onUpload: (newFileLocation: string, scaling: number) => void;
```

Comment:
Rename this to onSubmit to be consistent with HTML naming conventions in general.